src/plugins/jni/CMakeLists.txt

```diff
@@ -16,7 +16,7 @@ if (DEPENDENCY_PHASE)
 		set (DIRS ${JNI_INCLUDE_DIRS} ${JAVA_INCLUDE_PATH} ${JAVA_INCLUDE_PATH2} ${JAVA_AWT_INCLUDE_PATH})
 		set (LIBS ${JAVA_MAWT_LIBRARY} ${JNI_LIBRARIES} ${JAVA_AWT_LIBRARY} ${JAVA_JVM_LIBRARY}) # for check_symbol_exists
 		set (CMAKE_REQUIRED_INCLUDES ${DIRS})
-		safe_check_symbol_exists (JNI_VERSION_1_8 jni.h JNI_CORRECT_VERSION)
+		check_symbol_exists (JNI_VERSION_1_8 jni.h JNI_CORRECT_VERSION)
```

Comment:
The description for `safe_check_symbol_exists` reads:
https://github.com/ElektraInitiative/libelektra/blob/b266b2e8be83dc2d6218442c5d598d694d9fc264/cmake/Modules/SafeCheckSymbolExists.cmake#L4-L16

So it seems to me that `safe_check_symbol_exists` isn't working correctly, if this change actually fixed a bug.
Please check whether `safe_check_symbol_exists` has to be fixed as well.